### PR TITLE
[AD-184980] fix for switch statement

### PR DESCRIPTION
**Pull request by bmatthews-DFE**
ADO ticket: https://sfa-gov-uk.visualstudio.com/National%20Careers%20Service%20(NCS)/_workitems/edit/184980/

### CODE REVIEW

File: NCS.DSS.Outcomes/NCS.DSS.Outcomes.csproj [hunk 1]
```diff
@@ -23,6 +23,13 @@
     <PackageReference Include="Microsoft.Identity.Client" Version="4.49.0" />
```
Comment (bmatthews-DFE) [inline NCS.DSS.Outcomes/NCS.DSS.Outcomes.csproj:23]: What do these changes actually do?

**Review by JHUDSON-SMALL-DFE — APPROVED** [review]

File: NCS.DSS.Outcomes/NCS.DSS.Outcomes.csproj (hunk 1 above)
Comment (lschonauer-DFE) [reply NCS.DSS.Outcomes/NCS.DSS.Outcomes.csproj:23]: It ensures that Careers Progression outcomes can be claimed within 13 months of their 'Outcome Effective Date' (which is in line with the outcome rule/business process). Before this change, an error is raised if 12 months have passed which is incorrect.

Comment (bmatthews-DFE) [reply NCS.DSS.Outcomes/NCS.DSS.Outcomes.csproj:23]: I commented in the csproj file - do you need these changes to be committed?
@lschonauer-DFE

Comment (lschonauer-DFE) [reply NCS.DSS.Outcomes/NCS.DSS.Outcomes.csproj:23]: My mistake. I'm not sure to be honest. I think those changes were added automatically when I tried running this locally. They related to 'ignored' files so it should be OK.
Screenshot 2024-08-16 103308

Comment (bmatthews-DFE) [reply NCS.DSS.Outcomes/NCS.DSS.Outcomes.csproj:23]: Other blog posts appear to suggest this to be the case too: 
image

Comment (bmatthews-DFE) [reply NCS.DSS.Outcomes/NCS.DSS.Outcomes.csproj:23]: Will resolve @jpotter-DFE

**Pull request merged**